pkg/types/types.go

```diff
@@ -0,0 +1,40 @@
+package types
+
+type LandscapeList struct {
```

Comment:
I generated this myself. Since all of the Landscape tools are all in JS, there isn't really a hard schema they follow. 

Closest thing is here: https://github.com/cncf/landscapeapp/blob/9304e3a7771767785de8f4c13ef2913e7602114e/tools/removeQuotes.js#L48

The idea would be if the schema changed, we would catch that in the next PR since verify would fail.